### ISSUE TITLE
Don't open pianoroll when setting ghost notes

### ISSUE THREAD

**Issue by allejok96** [LMMS/lmms#6233]
Most of the time you'd set ghost from one pattern to use it in another pattern, so the workflow would be:
1. Right click pattern A.
2. Set as ghost.
3. Double click pattern B.

This is interrupted by the fact that 2. opens the piano roll in your face, so if it overlaps song editor you have to close it to proceed.

This should be a good first issue, probably just one line :)

**Comment by merrickclay** [comment]
I do think this is a good idea, however I think it's pretty standard in applications to open the window that is affected when a menu item is selected. It kind of notifies the user that the menu item they selected actually executed and worked properly.

Is there another way that we can provide visual feedback to the user when they set a pattern as ghost so it doesn't look like nothing occurred?

**Comment by allejok96** [comment]
Good point, feedback is good. We could add a notification, similar to when you try to glue notes when no notes are selected.
